ui/src/containers/Header/Header.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { mount } from 'enzyme';
+import { when } from 'mobx';
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Header from '.';
+import Search from '../../containers/Search';
+import UserProfile from '../UserProfile';
+import { FakeHub } from '../../api/testutil';
+import { createProviderAndStore } from '../../store/root';
+
+const TESTDATA_DIR = `src/store/testdata`;
+const api = new FakeHub(TESTDATA_DIR);
+const { Provider, root } = createProviderAndStore(api);
+
+describe('Header', () => {
```

Comment:
Can we please add a unit test to find a `login` component when the user is not authenticated